build.gradle

```diff
@@ -24,8 +24,10 @@ dependencies {
     implementation 'mysql:mysql-connector-java:8.0.28'
```

Comment:
to nie wywala błędu?

Reply:
Nie, ponieważ podczas uruchamiania testów implementowane są `testImplementation` dependencies.
Na dowód, pomyślne zakończenie workflow w [tym PR](https://github.com/Best-offer-finder/backend/pull/14), w którym przetestowano działanie testów.
Nadal nie jesteśmy w stanie sprawdzić, czy workflow się poprawnie uruchomi po buildzie.